search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -95,3 +105,88 @@
                        "concat humanized rows:" t3))))]
       (info "Create report " t4)
       (str string-writer))))
+
+(defn safe-generate-humanizers-report-csv
+  "This convenience function wraps the report generator in a try/catch."
+  [context]
+  (try
+    (generate-humanizers-report-csv context)
+    (catch Exception e
+      (warn (.getMessage e) "Returning empty report."))))
+
+(def report-cache-key
+  "The key used to store the humanizer report cache in the system cache map."
+  :report-cache)
```

Comment:
I was thinking that this might be used for more than just humanizer reports in the future ... and that we might want to move this code to someplace more general at that time ...

But I think your point is valid: it's here now, let's name it as it is, and we can update the name when we move it. I'll change it.